eos-s3/README.md

```diff
@@ -0,0 +1,31 @@
+# SymbiFlow Toolchain Examples for QuickLogic EOS S3
+
+1. `btn_counter` - simple 4-bit counter driving LEDs. The design targets the [EOS S3 FPGA](https://www.quicklogic.com/products/eos-s3/).
+
+## Setting up the toolchain
+
+```bash
+export INSTALL_DIR="/opt/symbiflow/eos-s3"
+bash conda_installer.sh -b -p $INSTALL_DIR/conda && rm conda_installer.sh
+source "$INSTALL_DIR/conda/etc/profile.d/conda.sh"
+conda env create -f eos-s3/environment.yml
+conda activate eos-s3
+wget -qO- https://quicklogic-my.sharepoint.com/:u:/p/kkumar/Eb7341Bq-XRAukVQ6oQ6PrgB-qdFbrsrPEON1yTa4krFSA?download=1 | tar -xJ -C $INSTALL_DIR
```

Comment:
These two steps highlight that the quicklogic install instructions are wrong. They should not be referencing sharepoint / kkumar not doing a copy.

Reply:
This commands where copied directly from hidden installation script of [quicklogic toolchain installator](https://github.com/QuickLogic-Corp/quicklogic-fpga-toolchain/releases/download/v1.1.0/Symbiflow_v1.1.0.gz.run). 

To not reference sharepoint / kkumar quicklogic archdef needs to be uploaded somewhere else. Probably this should be done by someone from QL.

Copy is fixing the problem with yosys not finding shared files. It is probably a bug in the quicklogic-yosys conda package. The same copy is done in the hidden installation script.
If we do not want to directly copy we need to wait for a fixed conda package or still use the official installation script.

Reply:
> This commands where copied directly from hidden installation script of [quicklogic toolchain installator](https://github.com/QuickLogic-Corp/quicklogic-fpga-toolchain/releases/download/v1.1.0/Symbiflow_v1.1.0.gz.run).
> 
> To not reference sharepoint / kkumar quicklogic archdef needs to be uploaded somewhere else. Probably this should be done by someone from QL.

This should be generated from CI automatically. Talk to @kgugala about who should own fixing this.

> Copy is fixing the problem with yosys not finding shared files. It is probably a bug in the quicklogic-yosys conda package. The same copy is done in the hidden installation script.
> If we do not want to directly copy we need a fixed conda package or still use the official installation script.

Please log bugs about these issues. Both the conda package and the official installation script probably need fixing.